src/app/modules/shared/components/presentational/icon-button/icon-button.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component, Input } from '@angular/core';
   standalone: true,
 })
 export class IconButtonComponent {
-  @Input() imageUrl!:String;
+  @Input() imageUrl!: String;
```

Comment:
Do not use class ```String```. For types try to use the primitive type ```string``` when writing in typescript